modules/mod_mailinglist/models/m_mailinglist.erl

```diff
@@ -202,7 +202,7 @@ recipient_delete1(RecipientProps, Quiet, Context) ->
 recipient_confirm(ConfirmKey, Context) ->
 	case z_db:q_row("select id, is_enabled, mailinglist_id from mailinglist_recipient where confirm_key = $1", [ConfirmKey], Context) of
 		{RecipientId, _IsEnabled, ListId} ->
-			NewConfirmKey = z_ids:id(20),
+			NewConfirmKey = binary_to_list(z_ids:id(20)),
 			z_db:q("update mailinglist_recipient set confirm_key = $2, is_enabled = true where confirm_key = $1", [ConfirmKey, NewConfirmKey], Context),
```

Comment:
I'll remove the conversion then.

I noticed that there is also a problem with the documentation, I'll fix that as well.